packages/components/select/src/select.vue

```diff
@@ -418,6 +420,7 @@ export default defineComponent({
       values: placements,
       default: 'bottom-start',
     },
+    ariaLabel: String,
```

Comment:
Do we need to add a default value, for example, `select`? 🤔

Reply:
I don't think so. Aria attributes shouldn't be forced